eventmesh-common/src/main/java/org/apache/eventmesh/common/Pair.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.eventmesh.common;
+
+public class Pair<Left, Right> {
+
+    private Left left;
+
+    private Right right;
```

Comment:
We have two `Pair` classes in our project:

- org.apache.eventmesh.common.Pair
- org.apache.eventmesh.runtime.common.Pair

The usages of the latter one can be unified to the former one.

>This comment is a copy of https://github.com/apache/eventmesh/pull/3446#discussion_r1450992783

Reply:
> We have two `Pair` classes in our project:
> 
> * org.apache.eventmesh.common.Pair
> * org.apache.eventmesh.runtime.common.Pair
> 
> The usages of the latter one can be unified to the former one.

I will remove org.apache.eventmesh.runtime.common.Pair. I think should use org.apache.eventmesh.common.Pair

Reply:
Fixed in https://github.com/apache/eventmesh/pull/4840/commits/1508e4c24c9d2d78b696b50e93f5bbf3b7dd1446